packages/integration/src/bitbucket/config.ts

```diff
@@ -32,6 +33,14 @@ export type BitbucketIntegrationConfig = {
    */
   host: string;
 
+  /**
+   * The baseUrl of this provider, e.g. "https://bitbucket.org", which is passed
+   * into the Bitbucket client.
+   *
+   * If no baseUrl is provided, it will default to https://${host}
+   */
+  baseUrl: string;
```

Comment:
I'm not completely settled on making this a required property of the config. I think making it optional is better in the long run since it can still be inferred, and we still have the ability to make use of fallbacks. 

@benjdlambert, do you have any thoughts on this?